R/plot_feature_importance.R

```diff
@@ -151,12 +151,16 @@ plot.feature_importance_explainer <- function(x, ..., max_vars = NULL, show_boxp
                    stat = "identity", fill = "#371ea3", color = "#371ea3", width = 0.25)
   }
 
+  if (!is.null(attr(x, "loss_name"))) {
+    y_lab <- paste(attr(x, "loss_name"), "loss after permutations")
```

Comment:
make sure that this works if there is a vector in attr(x, "loss_name").
Maybe use `attr(x, "loss_name")[1]`

Reply:
Done